.travis/stages/verify/check-links-and-yaml-syntax/check-links

```diff
@@ -5,6 +5,8 @@ set -eu
 URI_FILE=$(find . -type f -path "*/src/main/java/*" -name "UrlConstants.java")
 green="\e[32m"
 end="\e[0m"
+FAILURE_PREFIX="Failed with Code: "
+IGNORE_LIST=("https://www.axisandallies.org/forums")
```

Comment:
Would really love more insight about why A&A.org is failing from travis, have you been able to do any more research/investigation into this? Having a full status code and perhaps full output from curl, if not manually done, then printed by the script would seem helpful.

It's hard to know if the previous failure message was accurate and was not masking something a bit more benign.

Reply:
No Idea, I haven't been able to reach @djensen47 via GitHub.
My best guess atm is that the builds might exceeded some sort of threshold to we get blocked at a firewall level

Reply:
A comment in the code with an issue reference would be good documentation and something for someone to check if there is ever follow-up on this.

I'm otherwise surprised the request volume was ever noticeable enough for us to run into this situation. I would think the build verification check, being one-time, would have been a pittance compared to a single user browsing A&A.org